File: gdscript/src/main/java/gdscript/psi/GdTokenType.java
package gdscript.psi;

import com.intellij.psi.tree.IElementType;
import gdscript.GdLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class GdTokenType extends IElementType {

    public GdTokenType(@NotNull @NonNls String debugName) {
        super(debugName, GdLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "GdTokenType." + super.toString();
    }

}

File: gdscript/src/main/kotlin/gdscript/psi/GdTokenType.kt
package gdscript.psi

import com.intellij.psi.tree.IElementType
import com.jetbrains.rider.godot.community.gdscript.GdLanguage
import org.jetbrains.annotations.NonNls

class GdTokenType(debugName: @NonNls String) :
    IElementType(debugName, GdLanguage) {

    override fun toString(): String {
        return "GdTokenType." + super.toString()
    }

}